### PR TITLE
Added Tragikomedes's info (also automatically changed formatting).

### PR DESCRIPTION
**Pull request by Tragikomedia**
Self-explanatory title.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <h2 ><b> Hans</b> </h2>
     <div class="col-sm-4">
        <div class="thumbnail border rounded">
        <h2><b>Tragikomedes</b></h2>
-         <img src="https://i.pinimg.com/564x/40/fd/d0/40fdd0577aa6e53d759c97baa0e2e208.jpg" alt="Profile Pic" width="400" height="300">
+         <img src="img/tragikomedes.jpg" alt="Profile Pic" width="400" height="300">
```
Comment (hanskokx) [inline index.html:100]: The alt should be more descriptive of the individual picture.

File: mi-card.html [hunk 2]
```diff
@@ -52,7 +52,9 @@ <h1> Mi Cards </h1>
                 <div class="p-2">
                     <h4><b class="mi-title-text">Username: </b>Rift3000</h4>
                     <h4><b class="mi-title-text">Title: </b>Group-10 Team Lead/Project Manager</h4>
-                    <h4><b class="mi-title-text">What do I define as success: </b>Become a 'full' Flutter developer with the ability to build, deploy, and market apps. Be apart of a strong team that contributes to the flutter community and beyond.
+                    <h4><b class="mi-title-text">What do I define as success: </b>Become a 'full' Flutter developer with
+                        the ability to build, deploy, and market apps. Be apart of a strong team that contributes to the
```
Comment (hanskokx) [inline mi-card.html:56]: Why the extra whitespace?

File: index.html (hunk 1 above)
Comment (Tragikomedia) [reply index.html:100]: Everybody has the same alt for their profile pic, I thought it's the expected situation. I can change it if the standards are different.

File: mi-card.html (hunk 2 above)
Comment (Tragikomedia) [reply mi-card.html:56]: I used VS Code's autoformatter - it seems it doesn't like lines that are too long.

**Review by Goober0329 — APPROVED** [review]
I have the same question about the whitespace, otherwise approved.

File: mi-card.html (hunk 2 above)
Comment (hanskokx) [reply mi-card.html:56]: Not that big of a deal, but something we will have to address if it causes problems in the future.

File: mi-card.html [hunk 3]
```diff
@@ -61,29 +63,33 @@ <h4><b class="mi-title-text">What do I define as success: </b>Become a 'full' Fl
         <div class="col">
             <div class="form-check mt-2">
                 <label class="form-check-label" for="check1">
-          <input type="checkbox" class="form-check-input" id="check1" name="option1" value="something">Successfully
-          market an app
-        </label>
+                    <input type="checkbox" class="form-check-input" id="check1" name="option1"
```
Comment (hanskokx) [inline mi-card.html:64]: id needs to be unique.

File: mi-card.html [hunk 4]
```diff
@@ -61,29 +63,33 @@ <h4><b class="mi-title-text">What do I define as success: </b>Become a 'full' Fl
...
+                        value="something">Successfully
+                    market an app
+                </label>
             </div>
             <div class="form-check">
                 <label class="form-check-label" for="check2">
-          <input type="checkbox" class="form-check-input" id="check2" name="option2" value="something">Publish an app
-          to the AppStore and PlayStore
-        </label>
+                    <input type="checkbox" class="form-check-input" id="check2" name="option2" value="something">Publish
```
Comment (hanskokx) [inline mi-card.html]: Same here

File: mi-card.html [hunk 5]
```diff
@@ -61,29 +63,33 @@ <h4><b class="mi-title-text">What do I define as success: </b>Become a 'full' Fl
...
+                    to the AppStore and PlayStore
+                </label>
             </div>
 
             <div class="form-check">
                 <label class="form-check-label" for="check2">
-          <input type="checkbox" class="form-check-input" id="check3" name="option2" value="something">Master a
-          Flutter State Management
-        </label>
+                    <input type="checkbox" class="form-check-input" id="check3" name="option2" value="something">Master
```
Comment (hanskokx) [inline mi-card.html]: Etc

File: index.html (hunk 1 above)
Comment (Tragikomedia) [reply index.html]: Ok, I added more descriptive alts to both of my pictures

File: mi-card.html (hunk 2 above)
Comment (Tragikomedia) [reply mi-card.html]: I reverted text line breaks to their previous positions.

File: mi-card.html (hunk 3 above)
Comment (Tragikomedia) [reply mi-card.html:64]: I used the template that was left in the files, I can change it a bit later.

File: mi-card.html (hunk 5 above)
Comment (Tragikomedia) [reply mi-card.html]: I think the ID problem is more troublesome, it seems ids of all the cards of all the people are the same. I don't if everyone is ok with me changing it something else.

Comment (Tragikomedia) [reply mi-card.html]: Ok, I changed all the IDs to reflect a "Name-check1" pattern etc., it doesn't seem to have broken anything.

**Review by Goober0329 — APPROVED** [review]

**Review by hanskokx — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Rift3000) [reply index.html]: Yeah I left all the templates like that. Each says profile pic but in the spirit of good coding we can make it more descriptive

File: mi-card.html (hunk 3 above)
Comment (Rift3000) [reply mi-card.html]: Also my fault. Left the ids as such.

File: mi-card.html (hunk 5 above)
Comment (Rift3000) [reply mi-card.html]: Nice

**Pull request merged**